workflow-runtime/src/test/java/com/squareup/workflow1/internal/WorkflowRunnerTest.kt

```diff
@@ -230,7 +233,7 @@ internal class WorkflowRunnerTest {
     scope.cancel("foo")
 
     dispatcher.advanceUntilIdle()
-    assertTrue(output.isCancelled)
+    assertFalse(output.isActive)
```

Comment:
This PR is only a draft because of this change and the one above. There seems to be behavior change in the new coroutines library which seems quite scary. It seems wrong that these two tests started to fail.

Reply:
`output.isCanceled` is false even though the test calls `scope.cancel()`. Same as above, it's unclear to me why this changed.